integration_testing/features/admin/admin-bulk-export-users-classrooms.feature

```diff
@@ -7,8 +7,9 @@ Feature: Admin bulk export of users and classrooms
       And there are learners enrolled in classrooms and coaches assigned to them
 
   Scenario: Export a CSV file containing all users in the facility
-    When I click on "Export" button under *Import and export users* heading
+    When I click on *Export* button under *Import and export users* heading
```

Comment:
good to know *  has to be used always, I copied this format from a different feature file, so it seems we have some incosistency in them.

Reply:
It's not a big deal, but we started early on with the convention to enclose **all UI elements** inside the *`*`* chars, and not capitalize all letters even though they do appear capitalized in the Kolibri UI. 

https://github.com/learningequality/kolibri/blob/develop/integration_testing/features/README.md#ui-elements-conventions

I try to edit them for consistency when people contribute Gherkin scenarios, but might have missed some instances.

Reply:
:+1: